data/.wrglsync.yaml

```diff
@@ -318,9 +324,15 @@ repos:
     fileName: fuse/event_scott_pd.csv
     primaryKey:
       - event_uid
+<<<<<<< HEAD
+  event-tangipahoa-so:
+    fileName: fuse/event_tangipahoa_so.csv
+    primaryKey: 
+=======
   event-new-orleans-so:
     fileName: fuse/event_new_orleans_so.csv
     primaryKey:
+>>>>>>> 9240ec865f6bf2d1002ee37427730d551439f56e
```

Comment:
Looks like this merge conflict is left unresolved. Please read [this](https://git-scm.com/docs/git-merge#_how_conflicts_are_presented) to understand more about how conflicts are presented.

Reply:
Updated.